### PR TITLE
Site Settings: Show Toaster when Kit settings are saved

### PR DESCRIPTION
**Pull request by udidol**

### CODE REVIEW

**Review by kobizz — DISMISSED** [review]

File: core/kits/assets/js/manager.js [hunk 1]
```diff
@@ -37,7 +37,11 @@ export default class extends elementorModules.editor.utils.Module {
 			icon: 'eicon-global-settings',
 			title: elementor.translate( 'site_settings' ),
 			type: 'page',
-			callback: () => $e.route( 'panel/global/menu' ),
+			callback: () => {
+				$e.run( 'panel/global/open', {
+					route: $e.routes.getHistory( 'panel' ).reverse()[ 0 ].route,
```
Comment (matipojo) [inline core/kits/assets/js/manager.js:42]: Why route to the history item?

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/kits/assets/js/manager.js (hunk 1 above)
Comment (udidol) [reply core/kits/assets/js/manager.js:42]: @iNewLegend can explain, I did this part with his instruction

**Review by kobizz — APPROVED** [review]

File: core/kits/assets/js/manager.js (hunk 1 above)
Comment (iNewLegend) [reply core/kits/assets/js/manager.js:42]: @udidol, you can choose any route as you wish, maybe @matipojo means that the route to back to, can be static?

Comment (matipojo) [reply core/kits/assets/js/manager.js:42]: @udidol the `route` parameter is not relevant to here, please remove it.

Comment (udidol) [reply core/kits/assets/js/manager.js:42]: @matipojo @iNewLegend When I remove this parameter, the "Back to Editor" link does not bring me back to the last place I was in before entering Site Settings. With this parameter, it does redirect me back to where I came from (Editing a widget or the hamburger menu - the two places where you can access global settings).

**Review by matipojo — APPROVED** [review]

**Pull request merged**